package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "bluebird": "^3.3.1",
     "commander": "^2.9.0",
-    "cucumber": "^1.0.0",
+    "cucumber": "1.0.0",
```

Comment:
edit: nevermind, read the description. should we also fix our compatibility with the latest cucumber?